app/build.gradle

```diff
@@ -1,15 +1,14 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion '27.0.3'
-
+    compileSdkVersion 28
```

Comment:
As your SDK configuration is API level 28 so u need to set up with the one u have installed